scripts/qa_crowbarsetup.sh

```diff
@@ -2262,7 +2262,7 @@ function enable_ssl_generic()
             if [[ ! $(iscloudver 7plus) ]] ; then
                 return
             fi
-        ;;&
```

Comment:
@nicolasbock do you have any link about how this `;;&` is expected to work inside a `case`?

For me this is a +1